AK/StringUtils.h

```diff
@@ -7,6 +7,7 @@
 
 #pragma once
 
+#include <AK/Concepts.h>
```

Comment:
Changes look okay to me now except some rebasing got messed up - this belongs in the next commit